tests/tests

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
You may add `set -euo pipefail` for safer bash

Reply:
Done in 944bfb5a